.ci/azure/linux.yml

```diff
@@ -140,8 +143,21 @@ jobs:
       $(REPO_DIR)/install_build_dependencies.sh
       # Move jdk into contrib
       # 'clang' compiler is to check that samples can be built using it
-      sudo apt --assume-yes install openjdk-11-jdk clang
+      sudo apt --assume-yes install openjdk-11-jdk libbz2-dev clang
       # For opencv-python: python3-setuptools and pip upgrade
+      wget https://www.python.org/ftp/python/"$PYTHON_VERSION"/Python-"$PYTHON_VERSION".tgz
+      tar -xzf Python-"$PYTHON_VERSION".tgz
+      cd Python-"$PYTHON_VERSION" || exit
+      ./configure --enable-optimizations --prefix=/usr/local --enable-shared LDFLAGS="-Wl,-rpath /usr/local/lib"
```

Comment:
why do we need to build python from sources? I don't see any reason to test only python 3.10 in our CI.
Why do we need to change Azure? Why cannot we add 3.10 to Jenkins replacing 3.6?

Reply:
Discussion has been moved to internal channels.